content/docs/principles/build-for-people.md

```diff
@@ -1,18 +1,18 @@
 ---
 weight: 2
-description: We don't write code in a vacuum. We write code to solve problems for our customer, using systems used by their customers, all of which will be maintained by your teammates.
+description: We don't write code in a vacuum. We write code to solve problems for our customers using systems that their customers use, all of which will be maintained by our teammates.
 ---
 
 # Build for People
 
-We don't write code in a vacuum. We write code to solve problems for our customer, using systems used by their customers, all of which will be maintained by your teammates. We write code once but it is read many times and executed significantly more times.
+We don't write code in a vacuum. We write code to solve problems for our customers using systems that their customers use, all of which will be maintained by our teammates. We write code once but it is read many times and executed many times more than it is read.
 
 ## Solve the problem {#solve}
 
-You could write the most beautiful, most efficient, most poetic code you possibly can, but if it doesn't help your customer, it doesn't help anyone. Write great code and solve the problem.
+We write the most beautiful, most efficient, most poetic code you possibly can, but if it doesn't help our customer, it doesn't help anyone. We write great code and solve the problem.
```

Comment:
```suggestion
You could write the most beautiful, most efficient, most poetic code you possibly can, but if it doesn't help our customer, it doesn't help anyone. We write great code and solve the problem.
```

I am reverting part of the change here because the first part of the sentence is intentionally speculative and grandiose. It's not meant to be a fact.